p2p/discover/udp.go

```diff
@@ -268,6 +268,39 @@ func newUDP(c conn, ln *enode.LocalNode, cfg Config) (*Table, *udp, error) {
 	return udp.tab, udp, nil
 }
 
+// Quorum
+
+// ListenUDPExisting uses an existing table that listens for UDP packets on the connection.
+func ListenUDPExisting(c conn, ln *enode.LocalNode, cfg Config, tab *Table) (*Table, error) {
```

Comment:
Existing `ListenUDP()` and `newUDP()` functions are no longer referenced anywhere, is the plan to delete those?

Reply:
I hadn't planned to, in order to keep the diff from upstream smaller. `newUDP()` is still used in the upstream tests (`udp_test.go`).